lib/src/utils/private_views/instabug_private_view.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+import 'package:instabug_flutter/src/utils/private_views/private_views_manager.dart';
+import 'package:instabug_flutter/src/utils/private_views/visibility_detector/visibility_detector.dart';
+
+class InstabugPrivateView extends StatefulWidget {
+  final Widget child;
+
+  // ignore: prefer_const_constructors_in_immutables
+  InstabugPrivateView({required this.child}) : super(key: null);
```

Comment:
We can still have `InstabugPrivateView` take in a key like a normal widget and set it on its superclass, we don't have to set it as `null`. I believe we can turn it into a `const` constructor after that as well.

Reply:
@ahmedAlaaInstabug Same for the sliver widget.

Reply:
Making the constructor const prevents the VisibilityDetector from detecting changes in the view, as the parent widget remains unchanged and therefore doesn’t trigger a repaint.